bedrock/firefox/tests/test_base.py

```diff
@@ -275,15 +275,6 @@ def test_rv_prefix(self, render_mock):
         template = render_mock.call_args[0][1]
         eq_(template, ['firefox/australis/whatsnew.html'])
 
-    @override_settings(DEV=False)
-    def test_fx_australis_secure_redirect(self, render_mock):
-        """Should redirect to https: for 29.0."""
-        url = '/en-US/firefox/whatsnew/'
-        req = self.rf.get(url)
-        with patch.object(req, 'is_secure', return_value=False):
-            resp = self.view(req, version='29.0')
-        eq_(resp['location'], 'https://testserver' + url)
-
     @override_settings(DEV=True)
     def test_fx_australis_secure_redirect_not_dev(self, render_mock):
```

Comment:
`test_fx_australis_secure_redirect_secure` below might also be ok to remove now.